Sources/SpeziViews/Views/Text/MarkdownView.swift

```diff
@@ -95,14 +96,16 @@ public struct MarkdownView: View {
     /// Parses the incoming markdown and handles the view's error state management.
     /// - Parameters:
     ///   - markdown: A `Data` instance containing the markdown file in an utf8 representation.
-    ///
+    ///   Example: Data(
+    ///   " # This is a markdown example
+    ///    *This should be italiced* and **this bolded**").utf8
```

Comment:
If this is a code example, make sure to wrap them into markdown `swift` code block, as you did for the `TextContent´ type as well.
This makes sure that inline code appears with proper syntax highlighting. Make sure to properly format the code according to your swiftlint code style rules.